"Hello Microverse" project

alexpuente0 opened the pull request:
Created all files necessary for the project.

alexpuente0 committed:
Initial File Setup

Loaded Linters, ReadMe file and GitIgnore file

alexpuente0 committed:
Create index.html

Created the Index.html file for the project

alexpuente0 committed:
Create style.css

Created the Style.css file and changed the font color.

alexpuente0 committed:
Create test.md

Created the test for GitIgnore

alexpuente0 committed:
Update .gitignore

Updated GitIgnore to ignore Test.md

alexpuente0 committed:
Update .gitignore

Added additional information

alexpuente0 committed:
Delete test.md

Manually delete file to re-try gitIgnore

alexpuente0 committed:
Update .gitignore

Manually deleted test.md from repository and retried the gitIgnore method

alexpuente0 committed:
Update index.html

Updated charset and viewport

alexpuente0 committed:
Update style.css

Updated css file to sohw indentation and add spaces and semicolons

CollinsTatang reviewed (changes requested):
Hi @alexpuente0, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x] Please make sure your [linter](https://github.com/microverseinc/linters-config/tree/master/html-css) are working before you submit for the next review.
- [x] Kindly give a descriptive summary to your pull request before submitting for review.
- [x] Kindly give a descriptive pull request title.
- [x] Kindly give a descriptive commit message.
- [x] Kindly follow the microverse [Readme template](https://github.com/microverseinc/readme-template) to set up your readme file.




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

alexpuente0 committed:
Update style.css

Updated code to comply with Linters

alexpuente0 committed:
Update style.css

Added a new line of code to comply with StyleLint

alexpuente0 commented:
## "Hello Microverse" Project

On this project, we practiced the use of: 

-GitHub and Workflow
-Git Ignore 
-Linters
-ReadMe Files

alexpuente0 committed:
Update README.md

Updated the ReadMe file to add the "Getting Started", "Authors", "Contribution", "Support", "Acknowledgments", and "License" sections

RolandM99 reviewed (approved):
<h1>Status: Approve 🥇 ✔</h1>
Hi @alexpuente0 , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

- [ ] [ OPTIONAL ] Instead of creating a new comment while requested by the previous reviewer  to document and make descriptive your Pull Request, I think it's better to edit the previous comment you created for your PR by following the steps below: 
![image](https://user-images.githubusercontent.com/71874425/158481033-a1e8b96b-8bb7-4d18-87a9-4f60180e21c5.png)
- Click where the red arrow point,
- and then, click on Edit 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
